lib/uof_feed/messages/fixture_change.ex

```diff
@@ -0,0 +1,98 @@
+defmodule UofFeed.Messages.FixtureChange do
+  @moduledoc """
+  DataSchema representation of a FixtureChange message.
+
+  ### Field list:
+    - `start_time` - start time of the event,
+    - `product` - the ID of the producer that generated the message,
+    - `event_id` - the ID of the sport event,
+    - `timestamp` - the time the message was generated,
+    - `change_type` - fixture change type
+
+  ### Change type field mapping:
+    - 1 - `NEW`
+    - 2 - `DATE_TIME`
+    - 3 - `CANCELLED`
+    - 4 - `FORMAT`
+    - 5 - `COVERAGE`
+    - 6 - `PITCHER`
```

Comment:
Would be nice to have this as a proper enumerable, so that we can refer to fields as atoms, instead of shuffling magic numbers around. Not necessarily in this PR, though.

Reply:
Created https://github.com/esl/uof_feed/issues/3 for this.